### ISSUE TITLE
`force` is not very self-explaining

### ISSUE THREAD

**Issue by thomaseizinger** [sagiegurari/cargo-make#120]
#### Problem Description

The `force` argument of a task is not very self-explaining. I had to check the documentation on what it does. 

Maybe a name like `allow_failure` or `ignore_failure` could express the intent more clearly?

#### Code Example

````rust
[tasks.cleanup-cache]
allow_failure = true
script = ["..."]
````

**Comment by sagiegurari** [comment]
agree allow_failure is much better and self explanatory, I kinda reused the 'force' name from other tools I saw such as gruntjs.
unfortunately, changing this would be a real backward break and i'm not sure its important enough to justify it.

**Comment by sagiegurari** [comment]
fixing takes little time, understanding why takes longer and it would increase user frustration. I'll think about it. for now I'm keeping this item open.

**Comment by thomaseizinger** [comment]
That is why I added "if the error message is good"! Was thinking about something like `The attribute 'force' has been replaced with 'allow_failure'. Please update your configuration.`

Thanks for considering it! :)

**Comment by sagiegurari** [comment]
@thomaseizinger I decided to go for it. I renamed it to ignore_errors.
right now both attributes are supported and if force is used, it will have the error message you suggested.

**Comment by sagiegurari** [comment]
@thomaseizinger published 0.16.8 so closing this issue.
if there is anything else, please open a new issue.

**Comment by thomaseizinger** [comment]
Awesome!
Thank you for making `cargo-make` and putting so much effort in into it!
